schemas/context/namespace.description.md

```diff
@@ -0,0 +1,15 @@
+The namespace ensures first, second, and third party user and device identities can be used in customer engagement platforms.
+
+Namespaces are used to attain global uniqueness with user and device identities that originate in distributed, uncoordinated identity environments. 
+It can be thought of as an automatic prefix generator that can be used to append a unique prefix to other generators of identities that are only guaranteed to be unique in the scope of the individual generating system (the standard case for cookies and integer based identities).
```

Comment:
What can be thought of as a prefix generator? The namespace? Or the entity that generates the namespace? 

Again, I can only guess here: we have a number of systems that don't know each other that generate IDs. In order to keep the IDs apart, we concatenate the generated ID with an ID of the generator. 

What's not being made clear is:
1. does namespace describe the generator or the generated output
2. who creates IDs for generators?